json2html/jsonconv.py

```diff
@@ -78,88 +78,81 @@ def columnHeadersFromListOfDicts(self, inputtedJson):
                 -----------------------------
 
             @contributed by: @muellermichel
-        '''
+        """
 
```

Comment:
I recommend adding an exception here when inputted_json is not a list. Actually `inputted_json` would better be renamed to `input_list`. The old code would throw an exception in most cases because `len` wouldn't exist, however a graceful error would be better for code stability in later iterations.